### DISCUSSION TITLE
Community wants GitHub Mobile open source

### DISCUSSION THREAD

**Discussion by adriantabirta** [community/community#70048] [Mobile]
The app looks good, please make it open source 🙏

Just 🙏

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.134.2 (112648498)
Device: iPhone 8 (iOS 15.1)
</details>

**Comment by elisealix22** [comment]
Hi @adriantabirta , thank you for reaching out! This is something our team will continue to evaluate and it's good to know it's something you are interested in. Let us know if you have any other feature requests in the meantime.
